src/bulk-load.ts

```diff
@@ -508,6 +515,21 @@ class BulkLoad extends EventEmitter {
       addOptions.push('TABLOCK');
     }
 
+    if (this.bulkOptions.order.length) {
+      const orderColumns = [];
+
+      for (const column of this.bulkOptions.order) {
+        // Ensure column exists as a specified column, otherwise we ignore.
+        if (column.columnName && this.columnsByName[column.columnName] !== null) {
+          orderColumns.push(`${column.columnName} ${column.direction}`);
+        }
+      }
+
+      if (orderColumns.length) {
+        addOptions.push(`ORDER (${orderColumns.join(',')})`);
```

Comment:
Can we add a whitespace here? 😅 

```suggestion
        addOptions.push(`ORDER (${orderColumns.join(', ')})`);
```